tests/app/test_session.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from flask import url_for
+
+
+@pytest.mark.usefixtures("app")
+@pytest.mark.parametrize(
+    "todo, expected",
+    [
+        (
+            True,
+            "/sign-out?todo=True",
```

Comment:
another scenario with all `todo=true`

Reply:
I would keep it as it is as it always returns as `True` or `False` from the template.